### PR TITLE
Fix router not matching url

### PR DESCRIPTION
**Pull request by hwbllmnn**
If you have eg. a mapserver configured on the same host as Mapbender, matching with the Symfony router fails for URLs such as http://hostname/cgi-bin/mapserv in case mod_rewrite is configured to remove `app.php`. This PR fixes that by falling back on owsproxy in these cases.

### CODE REVIEW

**Comment by eSlider** [comment]
@swinkelmann, @AxxL can u please check this code in action and give some voice here? @hwbllmnn, thx!

**Comment by LazerTiberius** [comment]
I agree with @wg-rolf but this file needs some refactoring in general.

File: src/Mapbender/PrintBundle/Component/PrintService.php [hunk 1]
```diff
@@ -287,10 +287,25 @@ private function getImages($width, $height)
             $hostpath = $host . $parsed['path'];
             $pos      = strpos($hostpath, $this->urlHostPath);
             if ($pos === 0 && ($routeStr = substr($hostpath, strlen($this->urlHostPath))) !== false) {
-                $attributes = $this->container->get('router')->match($routeStr);
-                $gets       = array();
-                parse_str($parsed['query'], $gets);
-                $subRequest = new Request($gets, array(), $attributes, array(), array(), array(), '');
+                try {
```
Comment (atd-schubert) [inline src/Mapbender/PrintBundle/Component/PrintService.php]: This part repeats later...

File: src/Mapbender/PrintBundle/Component/PrintService.php [hunk 2]
```diff
@@ -580,10 +595,25 @@ private function addOverviewMap()
             $hostpath = $parsed['host'] . $parsed['path'];
             $pos      = strpos($hostpath, $this->urlHostPath);
             if ($pos === 0 && ($routeStr = substr($hostpath, strlen($this->urlHostPath))) !== false) {
-                $attributes = $this->container->get('router')->match($routeStr);
-                $gets       = array();
-                parse_str($parsed['query'], $gets);
-                $subRequest = new Request($gets, array(), $attributes, array(), array(), array(), '');
+                try {
```
Comment (atd-schubert) [inline src/Mapbender/PrintBundle/Component/PrintService.php]: This part repeats...

**Comment by LazerTiberius** [comment]
We should discuss here whether we need this fix for the next release or whether there is more room for additional improvements and refactoring but postpone it then to the 3.1 release?

**Comment by AxxL** [comment]
@atd-schubert We have your issues in mind but will merge it.

**Comment by werrolf** [comment]
Is there a problem with sending every request via owsproxy?
What's the motivation behind even trying to let the Symfony kernel handle these requests?

**Comment by simonseyock** [comment]
This branch is used in a project. I will merge it in another branch and then close again.

**Pull request closed without merging**